src/test/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnumUnitTest.java

```diff
@@ -144,37 +174,34 @@ public void testNonAlleleSpecificTwoAlts() {
                 .attribute(STRAND_BIAS_BY_SAMPLE_KEY, "1,0,4,5") // KCIBUL: why doesn't this work with: new int[]{2,0,6,0}  What is right?
                 .make();
 
-        builderA.attribute("AS_QUALapprox","|31|29|0")
-                .attribute("AS_VarDP","1|6|3|0")
-//                .attribute("BaseQRankSum","-0.572")
-                .attribute("DP","18")
-                .attribute("MQRankSum","0.696")
-                .attribute("QUALapprox","52")
-                .attribute("RAW_GT_COUNT","0,0,1")
-//                .attribute("RAW_MQ",".00")
-                .attribute("RAW_MQandDP","2808,18")
-                .attribute("ReadPosRankSum","-0.696")
-                .attribute("VarDP","10")
+        builderA.attribute(AS_RAW_QUAL_APPROX_KEY,"|31|29|0")
+                .attribute(AS_VARIANT_DEPTH_KEY,"1|6|3|0")
+                .attribute(DEPTH_KEY,"18")
+                .attribute(MAP_QUAL_RANK_SUM_KEY,"0.696")
+                .attribute(RAW_QUAL_APPROX_KEY,"52")
+                .attribute(RAW_MAPPING_QUALITY_WITH_DEPTH_KEY,"2808,18")
+                .attribute(READ_POS_RANK_SUM_KEY,"-0.696")
+                .attribute(VARIANT_DEPTH_KEY,"10")
                 .genotypes(Arrays.asList(g));
 
 
         VariantContext vc = builderA.make();
 
         Assert.assertEquals(VetFieldEnum.ref.getColumnValue(vc), "C");
         Assert.assertEquals(VetFieldEnum.alt.getColumnValue(vc), "CTTT,CTT");
-        Assert.assertEquals(VetFieldEnum.AS_RAW_MQ.getColumnValue(vc), "936|936|936");
-        Assert.assertEquals(VetFieldEnum.AS_RAW_MQRankSum.getColumnValue(vc), "0.696,1|0.696,1");
+        Assert.assertEquals(VetFieldEnum.AS_RAW_MQ.getColumnValue(vc), "2808|2808|2808");
```

Comment:
How are we going to calculate MQ here?  VarDP is just informative reads, which is going to inflate our numbers, but here I'm getting ~17, which can't be right.